internal/collector/vector/visitors.go

```diff
@@ -44,6 +45,11 @@ func CollectorVisitor(collectorContainer *corev1.Container, podSpec *corev1.PodS
 			Items:      []corev1.KeyToPath{{Key: RunVectorFile, Path: RunVectorFile}},
 			Optional:   utils.GetPtr(true),
 		}}},
+		corev1.Volume{Name: common.SecretDataReader, VolumeSource: corev1.VolumeSource{Secret: &corev1.SecretVolumeSource{
+			SecretName: resNames.ConfigMap,
+			Items:      []corev1.KeyToPath{{Key: SecretDataReaderScriptFile, Path: SecretDataReaderScriptFile}},
+			Optional:   utils.GetPtr(true),
```

Comment:
This isn't optional any longer right? Same is really true of the runscript.

Reply:
will check